seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/sink/JdbcSink.java

```diff
@@ -131,7 +131,10 @@ public SeaTunnelDataType<SeaTunnelRow> getConsumedType() {
 
     @Override
     public Optional<Serializer<JdbcAggregatedCommitInfo>> getAggregatedCommitInfoSerializer() {
-        return Optional.of(new DefaultSerializer<>());
+        if (jdbcSinkOptions.isExactlyOnce()) {
```

Comment:
Why change this? @ic4y PTAL

Reply:
> Why change this? @ic4y PTAL
This code is to adapt to the higher version of flink.If you don't need two-phase commit, you need to ensure that the serializer is also empty.
https://github.com/apache/flink/blob/a921a4d848e70f1569c5f20efb7b634bdc80de82/flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/operators/sink/CommitterOperatorFactory.java#L73